src/kt_member.h

```diff
@@ -0,0 +1,13 @@
+#ifndef GODOT_JVM_KT_MEMBER_H
+#define GODOT_JVM_KT_MEMBER_H
+
+
+#include <core/string_name.h>
+
+class KtMember {

```

Comment:
Do we need this? I.e will there be a time where we are going to return a `KtMember`?